### DISCUSSION TITLE
Can I disable the included wifi chip?

### DISCUSSION THREAD

**Discussion by kameo4242** [revk/xESP32-Faikout#106] [General]
I'm using a standard external "pre cloud" wifi for my Daikin but intend to switch to Faikins.
In both cases, is it recommended or not to disable the existing embedded Wifi chip once you have your new one plugged on the S21?

**Comment by revk** [comment]
Entirely up to you. I believe they work happily together on most systems. On mine it was a matter of swapping the old WiFi module for the Faikin, but if you have separate S21 they can both be together.

**Reply by ando2040** [reply]
My FTXM-R is working happily with both - in theory the family could have the simple Daikin interface and I could use Home Assistant. If you wanted to disable the Daikin wifi, I'm pretty certain you could just pull out the plug.

Mine has a wiring diagram inside the case, but even without it, would be easy to trace the cable from the wifi board to the S801 and disconnect.
![image](https://github.com/revk/ESP32-Faikin/assets/153862115/b9cabcfa-0ccc-4899-8bcf-ea45e78020b5)

**Comment by 44616E** [comment]
We have both running at the same time on ours XL premium units. But I am concerned if Daikin can push a firmware update to the AC that may impact the faikin. 

Does anyone know if the WiFi modules can update the ac firmware?

**Comment by trullock** [comment]
I've unplugged the stock wifi board and everything still works as far as i can tell

**Comment by asic79** [comment]
Just disabled the WiFi using the remote control: while the appliance is off, hold the powerful button until SP is displayed. Change to off using the temperature rocker. Hold the powerful button for ~2 secs. Press cancel on the remote. Done. 
If you need to re enable it, do the above but this time choose "3" instead of "off".

**Reply by wurmtal868** [reply]
Thanks, for reminding me :-)
I only disabled the internet access in my router to prevent the cloud connection.
By turning off the modules I was able to reclaim 6 IPs in my local WiFi without the use of a screwdriver. :-D

As a sidenote:
The use of the IR-Remote messed up the demand control of my multi split outdoor unit. A short disconnect from mains fixed this issue.
